.github/ISSUE_TEMPLATE/trac_ticket.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Trac Ticket
+about: Add a Trac Ticket to follow
+title: "Trac XXXXX: {Summary}"
```

Comment:
So far we've mostly used #XXXXX {Summary}, maybe we can stick with that?
```suggestion
title: "#XXXXX: {Summary}"
```

Reply:
I thought about that. I was worried that GH would try to auto-link to issue or PR numbers so this would be more clear at a glance. Thoughts?

Reply:
Makes sense, happy to go with this approach.